templates/prescription/general_edit.html

```diff
@@ -4,21 +4,42 @@
 
```

Comment:
I'll plan to review this script and the below script on the next review round (after all above issues are addressed). At that point will also starting testing this code.